### PR TITLE
Updating READMEs to have valid external links

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

File: sdk/communication/azure-communication-administration/README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Azure Communication Administration is used for managing users and tokens for Azure Communication Services.
 
-<!-- [Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
-| [Product documentation][azconfig_docs] -->
+[Source code][source] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][azconfig_docs]
```
Comment (DominikMe) [inline sdk/communication/azure-communication-administration/README.md:6]: I think we need to replace all azconfig_docs / appconfig links with links to acs

File: sdk/communication/azure-communication-sms/README.md [hunk 2]
```diff
@@ -121,5 +115,9 @@ Check out other client libraries for Azure Communication Services
 [coc]: https://opensource.microsoft.com/codeofconduct/
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+[azconfig_docs]: https://docs.microsoft.com/azure/azure-app-configuration
```
Comment (DominikMe) [inline sdk/communication/azure-communication-sms/README.md:118]: Here too, for exmple

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 3]
```diff
@@ -14,30 +14,25 @@
...
-        // application called COMMUNICATION_SERVICES_ENDPOINT.
-        String endpoint = System.getenv("COMMUNICATION_SERVICES_ENDPOINT");
-
-        // Retrieve the access key string for use with the application. The access key
-        // string is stored in an environment variable on the machine running the application 
-        // called COMMUNICATION_SERVICES_ACCESS_KEY.
-        String accessKey = System.getenv("COMMUNICATION_SERVICES_ACCESS_KEY");
+        // Your can find your endpoint and access token from your resource in the Azure Portal
+        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+        String accessToken = "SECRET";
```
Comment (DominikMe) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:28]: Let's rename this back to `accessKey`, it's not a token but the AccessKey from the portal

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 4]
```diff
@@ -14,30 +14,25 @@
...
 
         // Instantiate the http client
-        HttpClient httpClient = null; // Your HttpClient
-
+        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
```
Comment (DominikMe) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:31]: Thank you :-)

**Review by DominikMe — CHANGES_REQUESTED** [review]
Thanks for cleaning this up!

Found small things to comment on.

File: sdk/communication/azure-communication-sms/README.md (hunk 2 above)
Comment (DominikMe) [reply sdk/communication/azure-communication-sms/README.md:118]: Our docs are at
https://docs.microsoft.com/en-us/azure/communication-services/

Comment (minnieliu) [reply sdk/communication/azure-communication-sms/README.md]: Thank you, addressed.

**Comment by minnieliu** [comment]
> Thanks for cleaning this up!
> 
> Found small things to comment on.

Thank you, I've resolved all the comments

File: sdk/communication/azure-communication-chat/README.md [hunk 5]
```diff
@@ -75,9 +78,9 @@ String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
 NettyAsyncHttpClientBuilder httpClientBuilder = new NettyAsyncHttpClientBuilder();
 HttpClient httpClient = httpClientBuilder.build();
 
-// Your user access token retrieved from your trusted service
-String token = "SECRET";
-CommunicationUserCredential credential = new CommunicationUserCredential(token);
+// Your access key retrieved from your trusted service
+String accessKey = "SECRET";
+CommunicationUserCredential credential = new CommunicationUserCredential(accessKey);
```
Comment (DominikMe) [inline sdk/communication/azure-communication-chat/README.md:83]: `CommunicationUserCredential` actually takes a user access token, not the accessKey. This change here should be reverted, sorry about the confusion.

File: sdk/communication/azure-communication-chat/README.md [hunk 6]
```diff
@@ -382,3 +385,9 @@ Check out other client libraries for Azure communication service
 [coc]: https://opensource.microsoft.com/codeofconduct/
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+[product_docs]: https://docs.microsoft.com/azure/communication-services/
+[package]: https://search.maven.org/artifact/com.azure/azure-communication-chat
+[api_documentation]: https://aka.ms/java-docs
+[source]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/communication/azure-communication-chat/src
+
+
```
Comment (DominikMe) [inline sdk/communication/azure-communication-chat/README.md:393]: nit: extra new line

File: sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/ReadmeSamples.java [hunk 7]
```diff
@@ -45,9 +45,9 @@ public ChatClient createChatClient() {
         NettyAsyncHttpClientBuilder httpClientBuilder = new NettyAsyncHttpClientBuilder();
         HttpClient httpClient = httpClientBuilder.build();
 
-        // Your user access token retrieved from your trusted service
-        String token = "SECRET";
-        CommunicationUserCredential credential = new CommunicationUserCredential(token);
+        // Your user access key retrieved from your trusted service
+        String accessKey = "SECRET";
+        CommunicationUserCredential credential = new CommunicationUserCredential(accessKey);
```
Comment (DominikMe) [inline sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/ReadmeSamples.java:50]: Same here, Chat takes a token, not the access key.

(Look out for CommunicationUserCredential which always wraps a token)

**Review by DominikMe — CHANGES_REQUESTED** [review]
One change left :-)

Clarified accessKey vs user access tokens in the comments.

**Comment by DominikMe** [comment]
Build error:

```
[WARNING] Failed to resolve old artifacts: Failed to find a version of artifact 'com.azure:azure-communication-sms:jar:RELEASE' that would correspond to an expression '^\d+\.\d+\.\d+$'. The versions found were: [1.0.0-alpha.20200928.0, 1.0.0-alpha.20200925.0]. The API analysis will proceed comparing the new archives against an empty archive.
```

Guess we need to reference the beta version?

**Comment by minnieliu** [comment]
> Build error:
> 
> ```
> [WARNING] Failed to resolve old artifacts: Failed to find a version of artifact 'com.azure:azure-communication-sms:jar:RELEASE' that would correspond to an expression '^\d+\.\d+\.\d+$'. The versions found were: [1.0.0-alpha.20200928.0, 1.0.0-alpha.20200925.0]. The API analysis will proceed comparing the new archives against an empty archive.
> ```
> 
> Guess we need to reference the beta version?
Its actually an issue with verifying the readme, I will look into it :) 
(verify-readme-codesnippet) on project azure-communication-chat: Command execution failed. Process exited with an error: 1

File: sdk/communication/azure-communication-chat/README.md (hunk 5 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-chat/README.md:83]: Oh whoops that was my bad, will revert in next commit!

**Review by DominikMe — APPROVED** [review]

**Pull request merged**